### DISCUSSION TITLE
Taskbar Icon Notification Badge - WhatsApp

### DISCUSSION THREAD

**Discussion by carlosercf** [valinet/ExplorerPatcher#3420] [Q&A] [answered]
Hi,

I have enabled Whatsapp to show a badge on its taskbar icon but it never shows when I'm using Explorer Patcher. Is it possible to show the notifications I receive on the taskbar as it appears when using the normal windows 11 taskbar? I never see new messages from Microsot Teams or Outlook as well. I use small icons.
I've attached a picture of the configuration on WhatsApp that enables the taskbar notification badge whenever I receive a new notification.

![image](https://github.com/valinet/ExplorerPatcher/assets/14218028/7b99c45a-57bb-4c05-a84c-62121988351d)

**Comment by Amrsatrio** [comment]
Do you have "Show badges on taskbar apps" enabled in Taskbar settings in the Settings app?

**Reply by carlosercf** [reply]
Its actually greyed out because I have "small icons" enabled. I think that windows-wise it is not possible to have badges on icons when you are using the small icons feature. :(

**Reply by Amrsatrio** [reply, marked as answer]
Yeah, unfortunately, it's not possible... But anyways in the future I may see if it is viable to have badges even on small taskbar on the reimplemented taskbar.

**Reply by carlosercf** [reply]
Thanks for the support anyways!
